include/itkCudaImageFromImageFilter.hxx

```diff
@@ -0,0 +1,36 @@
+/*=========================================================================
+ *
+ *  Copyright NumFOCUS
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         https://www.apache.org/licenses/LICENSE-2.0.txt
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ *=========================================================================*/
+#ifndef itkCudaImageFromImageFilter_hxx
+#define itkCudaImageFromImageFilter_hxx
+
+namespace itk
+{
+
+template <typename TInputImage>
+void CudaImageFromImageFilter<TInputImage>::GenerateData()
+{
+  const auto * in = this->GetInput();
+  auto *       out = this->GetOutput();
+  out->SetBufferedRegion(in->GetBufferedRegion());
+  out->SetRequestedRegion(in->GetRequestedRegion());
```

Comment:
Is it clear to you why you need to set the requested region here? It's not obvious to me why it would be different than the one that was propagated in `GenerateOutputRequestedRegion` but I might have missed something.